mpv.rb

```diff
@@ -0,0 +1,58 @@
+class Mpv < Formula
+  # based upon the last version of homebrew-core/Formula/mpv.rb at https://github.com/Homebrew/homebrew-core/commit/41444d526c40b93069b7f0c5414539deb0534179
+  desc "Media player based on MPlayer and mplayer2"
+  homepage "https://mpv.io"
+  url "https://github.com/privatezero/mpv/archive/2019-10-11.tar.gz"
```

Comment:
Should we store the resource here on AMIA Open Source?

Reply:
Yeah - I figured that once this formula got finalized that I would transfer the forked repo over to amiaos and make a new updated release